.arduino-ci.yml

```diff
@@ -0,0 +1,32 @@
+platforms:
+  rpipico:
+    board: rp2040:rp2040:rpipico
+    package: rp2040:rp2040
+    gcc:
+      features:
+      defines:
+        - ARDUINO_ARCH_RP2040
+      warnings:
+      flags:
+
+packages:
+  rp2040:rp2040:
+    url: https://github.com/earlephilhower/arduino-pico/releases/download/global/package_rp2040_index.json
```

Comment:
I see that `esp32` is selected down below. In this spirit, I am wondering why any references to the Pico are needed?

Reply:
These rpipico lines are not needed per se, however it would allow you to include it in a future test. 
Pico is not supported directly in the core Arduino build. 
This is what is needed to get the build working with an rpipico compiler.